src/main/java/org/nantipov/utils/wordhugopress/config/job/convertposts/ConvertPostsReaderConfig.java

```diff
@@ -30,14 +30,13 @@
     static final String BEAN_NAME = "postsReader";
 
     private final SourcesSettings sourcesSettings;
+ 
+    static String SQLQueryPost;
+    static String TablePrefix;
```

Comment:
Considering other comments, declaration of these variables might not be required.